### PR TITLE
fix(github): various unthemed elements

### LINKED ISSUE #202: Issues with new github tree explorer interface
### Is there an existing issue outlining your problem?

- [X] I have searched the existing issues and they do not solve my problem.

### What userstyles are you seeing the problem on?

lbl:github

### Describe your problem.

Mocha (Pink accent) theme. Github theme is light default, untouched.

### Attach screenshots.

image

### What browser(s) are you seeing the problem on?

LibreWolf 117.0-1

### Any additional comments?

_No response_

### LINKED ISSUE #302: Unthemed elements in the New GitHub repository overview
### Is there an existing issue outlining your problem?

- [X] I have searched the existing issues and they do not solve my problem.

### What userstyles are you seeing the problem on?

lbl:github

### Describe your problem.

When enabling the new repository overview from GitHub's "Feature Preview" settings, there are several elements that aren't themed.

### Attach screenshots.

image

### What browser(s) are you seeing the problem on?

Firefox Developer Edition 120.0b8

### Any additional comments?

_No response_

### PR DESCRIPTION
**Pull request by uncenter**
## 🔧 What does this fix? 🔧

Fixes various unthemed things.
- Closes #202.
- Closes #302.

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

**Comment by uncenter** [comment]
Self-reminder to do the commits page as well:
Screenshot 2024-02-03 at 21 41 27 (Arc)

**Comment by uncenter** [comment]
I'm going to try to fix *everything*. All of these: https://github.com/catppuccin/userstyles/issues?q=is:issue+is:open+sort:updated-desc+label:github (hopefully)

**Comment by GenShibe** [comment]
> I'm going to try to fix _everything_. All of these: https://github.com/catppuccin/userstyles/issues?q=is:issue+is:open+sort:updated-desc+label:github (hopefully)

i believe in you uncenter <3

File: styles/github/catppuccin.user.css [hunk 1]
```diff
@@ -386,87 +384,145 @@
...
+          background-color: @surface1;
         }
       }
+      .eJLRZa {
+        border-color: @surface1;
+      }
+    }
+    .Text-sc-17v1xeu-0.gPDEWA {
+      color: @subtext1;
+    }
```
Comment (uncenter) [inline styles/github/catppuccin.user.css:516]: Next order of business is trying to figure out what @isabelroses was styling here and add comments explaining it... send help.

Comment (isabelroses) [reply styles/github/catppuccin.user.css]: its for this 
image

**Comment by uncenter** [comment]
Couldn't theme everything. :cry:

**Review by isabelroses — APPROVED** [review]

**Pull request merged**